### ISSUE TITLE
Bevy 0.10 not happy with ActionState in Query<>

### ISSUE THREAD

**Issue by torvalamo** [Leafwing-Studios/leafwing-input-manager#324]
## Version

leafwing 0.8.0
bevy 0.10.0

## What you did

    fn camera_controls(
        camera_actions: Query<&ActionState<CameraAction>>,
        ...

## What actually happened

    camera_actions: Query<&ActionState<CameraAction>>,                             
         |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `bevy::prelude::Component` is not implemented for `ActionState<CameraAction>`
    = note: required for `&ActionState<CameraAction>` to implement `WorldQuery`
    = note: required by a bound in `Query`                                                     
    --> /Users/.../bevy_ecs-0.10.0/src/system/query.rs:276:37                                                                      
     |                                                                                    
     | pub struct Query<'world, 'state, Q: WorldQuery, F: ReadOnlyWorldQuery = ()> {      
     |                                 ^^^^^^^^^^ required by this bound in `Query`

## Additional information

I'm not a rust expert, but I did some investigation.

It seems Component has been changed (some added field(s) and behaviors), so since leafwing wants to use a lower version of bevy, it implements an unrecognizable Component.

Not sure if you can just bump the bevy version (or does cargo allow version ranges like npm?), but it probably wouldn't be too much cleanup. I might try it myself, but I don't trust my rust skills enough to submit any pull requests.

**Comment by alice-i-cecile** [comment]
Yep, we need to merge #317 and then you can use the released crate there. Should be ready momentarily :)
